students/wachulec_maria/lesson_01_basics/tens_digit.py

```diff
@@ -0,0 +1,3 @@
+number = input("Give me number: ")
+number = '00' + number
+print(number[-2])
```

Comment:
Relying so much on slicing strings is not the best idea. It would be preferred to do some operations on integers.